openmetadata-ui/src/main/resources/ui/src/styles/temp.css

```diff
@@ -729,7 +729,7 @@ body .list-option.rdw-option-active {
 }
 
 .slick-dots li button:before {
-  font-size: 24px !important;
+  font-size: 12px !important;
```

Comment:
@louie007 , any reason for this change?

Reply:
Hi Sachin, the font-size was incorrectly specified to 24px

Reply:
Hi Sachin, the font-size was incorrectly specified to 24px because the "slick" font family was never loaded correctly before. When we go check the slick carousel source code (https://github.com/kenwheeler/slick) and online demo (http://kenwheeler.github.io/slick/), we can see the original slick dot size was just 6px.

Reply:
Moreover, a minimum font-size of 12px is set for Chrome users with some UI locales (languages) - e.g. Chinese, Korean, Japanese, Thai etc. If however, the font-size for any content on your website is set to a value that is less than 12px, it will automatically be resized to 12px.

Reply:
To keep consistency of UI, let's set the font-size to 12px. This will make the dot size identical among whatever language.